notebooks/community/vision_ai/video_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
marked as resolved to discuss this in another comment.